geodiff/cmake/FindPostgres.cmake

```diff
@@ -0,0 +1,134 @@
+# Find PostgreSQL
```

Comment:
why the default one is not good enough? https://cmake.org/cmake/help/v3.0/module/FindPostgreSQL.html

I would say that if the default one is not good enough, user can always manually set POSTGRESQL_* vars on command line?

Reply:
This one (copied from QGIS source code) works well and it can be used for all our platforms without any adjustments. The default one from CMake didn't even work out of the box in my ubuntu and it would need extra adjustments for all other platforms too...